src/maintenance/performance.rst

```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
```

Comment:
I suggest rewording to: "MochiWeb handles CouchDB requests."

The current wording is not incorrect, but is a bit awkward in native English.